<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGLWindow</class>
 <widget class="QMainWindow" name="MyGLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fronti√®res</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="MyGLScreen" name="screen" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_scene"/>
    <addaction name="action_Save_scene"/>
    <addaction name="separator"/>
    <addaction name="action_Load_clouds_defaults"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_sample"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
   </widget>
   <widget class="QMenu" name="menu_OSC">
    <property name="title">
     <string>&amp;OSC</string>
    </property>
    <addaction name="action_Control"/>
   </widget>
   <widget class="QMenu" name="menu_Midi">
    <property name="title">
     <string>&amp;Midi</string>
    </property>
    <addaction name="action_Combi"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Midi"/>
   <addaction name="menu_OSC"/>
  </widget>
  <action name="action_Load_scene">
   <property name="text">
    <string>&amp;Load scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_scene">
   <property name="text">
    <string>&amp;Save scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Add_sample">
   <property name="text">
    <string>&amp;Add samples...</string>
   </property>
  </action>
  <action name="action_Load_clouds_defaults">
   <property name="text">
    <string>Load &amp;clouds defaults</string>
   </property>
  </action>
  <action name="action_Start_controller">
   <property name="text">
    <string>&amp;Start controller</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_Control">
   <property name="text">
    <string>&amp;Control</string>
   </property>
  </action>
  <action name="action_Combi">
   <property name="text">
    <string>&amp;Combinations</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLScreen</class>
   <extends>QWidget</extends>
   <header>interface/MyGLWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
